### PR TITLE
Fixing the ZK prover configuration example

### PR DESCRIPTION
**Pull request by ii-cruz**
## What's in this pull request?
A simple fix of the ZK prover configuration example.

## Pull request checklist

- [x] All tests pass. The project builds and runs.
- [x] I have resolved any merge conflicts.
- [x] I have resolved all `clippy` and `rustfmt` warnings.

### CODE REVIEW

**Review by jsdanielh — APPROVED** [review]

File: lib/src/config/config_file/client.example.toml [hunk 1]
```diff
@@ -19,15 +19,10 @@
 # Multiple addresses can be specified.
 #
 # If `advertised_addresses` is not used, these addresses will also be advertised to peers.
-listen_addresses = [
-  "/ip4/0.0.0.0/tcp/8443/ws",
-  "/ip6/::/tcp/8443/ws",
-]
+listen_addresses = ["/ip4/0.0.0.0/tcp/8443/ws", "/ip6/::/tcp/8443/ws"]
```
Comment (sisou) [inline lib/src/config/config_file/client.example.toml]: Please don't change this formatting.

File: lib/src/config/config_file/client.example.toml [hunk 2]
```diff
@@ -19,15 +19,10 @@
...
 
 # Seed nodes to connect to when the node starts up. Multiple addresses can be specified.
-seed_nodes = [
-  { address = "/dns4/seed1.pos.nimiq-testnet.com/tcp/8443/wss" },
-]
+seed_nodes = [{ address = "/dns4/seed1.pos.nimiq-testnet.com/tcp/8443/wss" }]
```
Comment (sisou) [inline lib/src/config/config_file/client.example.toml]: Please don't change this formatting.

**Review by sisou — CHANGES_REQUESTED** [review]

File: lib/src/config/config_file/client.example.toml (hunk 1 above)
Comment (Eligioo) [reply lib/src/config/config_file/client.example.toml]: Can you share why?

Comment (sisou) [reply lib/src/config/config_file/client.example.toml]: It's less readable, usable and extendable. Is this auto-formatted or why does this show up in @ii-cruz's PRs always?

Comment (ii-cruz) [reply lib/src/config/config_file/client.example.toml]: Yes, this is the default formating.

File: lib/src/config/config_file/client.example.toml (hunk 2 above)
Comment (ii-cruz) [reply lib/src/config/config_file/client.example.toml]: I put it back to the previous formating.

**Review by sisou — APPROVED** [review]

**Pull request merged**